lib/mercury/mercury.rb

```diff
@@ -216,9 +216,18 @@ def dispatch_publisher_confirm(basic_ack)
   def make_received_message(payload, metadata, work_queue_name: nil)
     msg = ReceivedMessage.new(read(payload), metadata, self, work_queue_name: work_queue_name)
     Logatron.msg_id = msg.headers[LOGATRAON_MSG_ID_HEADER]
+    Logatron.site = get_message_site(msg.content)
     msg
   end
 
+  def get_message_site(content)
+    content['organization'] ||
+      content['customer'] ||
+      content['site'] ||
+      content.fetch('request', {})['customer'] ||
+      '-'
```

Comment:
This is a rabbit stew concern, not mercury. It would be a mercury concern if it had to be passed along like msg_id is.